house/tests/house_tests.py

```diff
@@ -58,3 +69,32 @@ def test_delete_house(self, generate_house):
         generate_house.save()
         generate_house.delete()
         assert generate_house not in House.objects.all()
+
+
+@pytest.mark.django_db
+class TestMyHouseViews:
+    def test_mine_page_after_login_views_templates(self):
+        try:
+            get_template(MINE_EXPENSES_TITLE_ROUTE)
+        except TemplateDoesNotExist:
+            assert False, f"Template {MINE_EXPENSES_TITLE_ROUTE} does not exist"
+
+        try:
+            get_template(MINE_HOUSE_TABLE_ROUTE)
+        except TemplateDoesNotExist:
+            assert False, f"Template {MINE_HOUSE_TABLE_ROUTE} does not exist"
+
+        try:
+            get_template(MINE_HOUSE_TABLE_TITLE_ROUTE)
+        except TemplateDoesNotExist:
+            assert False, f"Template {MINE_HOUSE_TABLE_TITLE_ROUTE} does not exist"
+
+        try:
+            get_template(MINE_MINE_PAGE_ROUTE)
+        except TemplateDoesNotExist:
+            assert False, f"Template {MINE_MINE_PAGE_ROUTE} does not exist"
```

Comment:
Same here like the comment in the const URL

Reply:
I answered you in the first response